traveler-swift-core/TravelerKit/Models/CancellationError.swift

```diff
@@ -11,4 +11,6 @@ import Foundation
 public enum CancellationError: Error {
     case expiredQuote
     case notCancellable
+    case noReasonSelected
+    case noExplanationProvided
```

Comment:
`explanationRequired` is slightly better because it doesnt imply that all cancellations will fail if they dont provide a reason